src/plugins/intel_gpu/src/plugin/ops/random_uniform.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "cldnn_program.h"
+#include "cldnn_common_utils.h"
+#include "ngraph/op/random_uniform.hpp"
+#include "cldnn/primitives/random_uniform.hpp"
+
+
+namespace CLDNNPlugin {
+
+namespace {
+
+void CreateRandomUniformOp(Program &p, const std::shared_ptr<ngraph::op::v8::RandomUniform> &op) {
+    auto input_primitives = p.GetInputPrimitiveIDs(op);
+    auto output_shape = CldnnTensorFromIEDims(op->get_output_shape(0));
+    cldnn::format outputFormat = DefaultFormatForDims(op->get_output_shape(0).size());
```

Comment:
Maybe dedicated variable for get_output_shape(0) value since it is called twice?

Reply:
done